src/transformState.js

```diff
@@ -6,6 +6,31 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const key in actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
![image](https://user-images.githubusercontent.com/96409563/227860363-339701d4-150a-42bf-b32e-d45fa7afc8ab.png)
![image](https://user-images.githubusercontent.com/96409563/227860391-8e35cd11-a0c3-4c54-8ea1-6b559c7346d7.png)